### PR TITLE
Update translations using inlang

### PR DESCRIPTION
**Pull request by mbaraa**
INLANG

Add missing Arabic [ar-SA] translations, and fixed some inaccurate translations.

### CODE REVIEW

**Review by richardshiue — APPROVED** [review]

**Comment by richardshiue** [comment]
Hey there @mbaraa, thanks for contributing. Please merge upstream into your branch and resolve the conflicts

**Comment by mbaraa** [comment]
hey @richardshiue, I resolved the conflicts with upstream.

File: frontend/resources/translations/en.json [hunk 1]
```diff
@@ -1007,12 +983,7 @@
     }
   },
   "datePicker": {
-    "dateTimeFormatTooltip": "Change the date and time format in settings",
-    "dateFormat": "Date format",
-    "includeTime": "Include time",
-    "isRange": "End date",
-    "timeFormat": "Time format",
-    "clearDate": "Clear date"
+    "dateTimeFormatTooltip": "Change the date and time format in settings"
```
[the remark this answers is not in the document]
Comment (mbaraa) [reply frontend/resources/translations/en.json:986]: yeah sure.

[the remark this answers is not in the document]
Comment (mbaraa) [reply frontend/resources/translations/en.json]: done.

**Review by richardshiue — APPROVED** [review]
LGTM, I'll merge after the tests pass

**Comment by mbaraa** [comment]
Is the commit's prefix a big deal? 👀

**Comment by richardshiue** [comment]
> Is the commit message's prefix a big deal? 👀

No, not really. Maintainers can edit the commit message while merging the PR. But some don't like to see the red x lol

**Comment by mbaraa** [comment]
> > Is the commit message's prefix a big deal? 👀
> 
> No, not really. Maintainers can edit the commit message while merging the PR. But some don't like to see the red x lol

fair enough.

**Pull request merged**